### PR TITLE
Factory protected disk, Separated Chemicals in code(not it normal game)

### PR DESCRIPTION
**Pull request by botka4aet**
## What Does This PR Do
- Return Separated Chemicals - https://github.com/ParadiseSS13/Paradise/pull/16716
- Strange seeds can't get Separated Chemicals trait - new var and check
- Disks with recorded genes. Factory protected(you can't overwrite it)

## Why It's Good For The Game
- Separated Chemicals trait is still in code and game but players can't get them normaly(read "every round")
- Admins can spawn disk with this trait and add to traders stock. (Or even add low spawn chance for trader stock by default)
- Factory disk - full rewrite protect. But it can still be stolen(traitor target if disk exists?)

## Images of changes
image

## Changelog
:cl:
add: Factory plant disk - full rewrite protect
tweak: Strange plants can`t get prohibited traits
tweak: Separated Chemicals trait can only be obtained through admin panel
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
> Return Separated Chemicals - Removes Separated Chemicals #16716

From Code Of Conduct:
`Reactionary revert PRs are not tolerated under any circumstances.`

---

> tweak: Tweaked a few things

**far** too little detail in the changelog




TLDR: Its a no from me

**Comment by nukularpower** [comment]
yes please

**Comment by botka4aet** [comment]
> > tweak: Tweaked a few things
> 
> **far** too little detail in the changelog

Sorry, forgot to fill this

**Comment by matttheficus** [comment]
Botany doesnt need their powergame-combo back. You can still make warcrime tomatoes, thats plenty.

**Comment by S34NW** [comment]
I've still seen plenty of horrific warcrime fruit after the separated chemical removal. Not sure why you want to bring them back and make the situation even worse.

**Comment by botka4aet** [comment]
You can't obtain it without admin.
Admin can use it for event.
No warcrimes without permisstion.
I don't see any problem.

Any way asked @Fox-McCloud at discord. If mind - just cut all trait code. Disks and blocked strange seed trait still a thing.

PS. It's sad when some of the possibilities are cut out completely. Wanna see xenoarchaeology

**Comment by Fox-McCloud** [comment]
>From Code Of Conduct:
Reactionary revert PRs are not tolerated under any circumstances.
